app/views/template_invocations/_output_line.html.erb

```diff
@@ -0,0 +1,8 @@
+<% @_line_counter ||= local_assigns[:line_counter] || 0 %>
```

Comment:
Because I need to keep the value between `_output_line` renders, local one would always start from 0. Unfortunately we can't reuse built-in counter because some lines contain multiple rows...